### PR TITLE
Cp1605 omnibus avatar uploading doesnt work

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes CartoDB/cartodb-platform#1605.

@santisaez, @javisantana, @xavijam CR this, please.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
Retest this, please.

**Comment by javisantana** [comment]
:es:

File: app/models/asset.rb [hunk 1]
```diff
@@ -146,4 +150,18 @@ def s3_bucket
...
+  def chmod_mode
+    # Example in case asset kind should change mode
+    # kind == KIND_ORG_AVATAR ? 0644 : nil
+    0644
+  end
+
+  def asset_protocol
+    # Avatars without protocol to allow the browser pick http/https.
+    # Other assets with http, because, for example, tiler needs access to landmark images.
+    kind == KIND_ORG_AVATAR ? '' : 'http:'
```
Comment (javisantana) [inline app/models/asset.rb]: what is KIND_ORG_AVATAR ?

Comment (juanignaciosl) [inline app/models/asset.rb]: It's a constant defined in this file and used for avatars (not only organization avatars).

**Pull request merged**